### PR TITLE
ponoko instructions and file

### PR DESCRIPTION
**Pull request by pchinjr**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+#DS_Store
+.DS_Store
```
Comment (lynnaloo) [inline .gitignore]: .DS_Store!! No......!

**Comment by lynnaloo** [comment]
@pchinjr if you remove that DS_Store file, I'll pull this in

**Review by pchinjr — COMMENTED** [review]
ah ok, will remove it

**Comment by pchinjr** [comment]
ok, removed the ds store file. thanks!

**Pull request merged**